README.md

```diff
@@ -1,14 +1,14 @@
 # ![Microsoft Bot Framework Composer](./docs/media/gh-banner.png)
 
-# Microsoft Bot Framework Composer [PREVIEW]
+# Microsoft Bot Framework Composer
 
 [![Build Status](https://github.com/microsoft/BotFramework-Composer/workflows/Composer%20CI/badge.svg?branch=master)](https://github.com/microsoft/BotFramework-Composer/actions?query=branch%3Amaster)
 [![Coverage Status](https://coveralls.io/repos/github/microsoft/BotFramework-Composer/badge.svg?branch=master)](https://coveralls.io/github/microsoft/BotFramework-Composer?branch=master)
 [![Total alerts](https://img.shields.io/lgtm/alerts/g/microsoft/BotFramework-Composer.svg?logo=lgtm&logoWidth=18)](https://lgtm.com/projects/g/microsoft/BotFramework-Composer/alerts/)
 
 ## Overview
 
-Bot Framework Composer is an integrated development tool for developers and multi-disciplinary teams to build bots and conversational experiences with the Microsoft Bot Framework. Within this tool, you'll find everything you need to build a sophisticated conversational experience.
+Bot Framework Composer is an open-source, visual authoring canvas for developers and multi-disciplinary to design and build conversational experiences with Language Understanding and QnA Maker, and a sophisticated composition of bot replies (Language Generation). Within this tool, you'll have everything you need to build a sophisticated conversational experience.
```

Comment:
this sentence doesn't read correctly. did you mean to remove "teams"?

Reply:
OMG thanks for catching this - it's wrong on the website too!!


Bot Framework Composer is an open-source, visual authoring canvas for developers and multi-disciplinary teams to design and build conversational experiences with Language Understanding, QnA Maker and sophisticated composition of bot replies (Language Generation). 
Within this tool, you'll have everything you need to build a conversational experiences.